test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -339,6 +359,12 @@ public void forceCompact(String keyspace, String table)
         });
     }
 
+    @Override
+    public List<Throwable> getUncaughtExceptions()
```

Comment:
this looks like a breaking change in 0.0.5, so had to add it to compile

Reply:
FYI, there is an unmerged commit https://github.com/krummas/cassandra/commit/7cec7bd563fac3089e45fec89ac3f493683f0a85 that implements the new interface. (whoever merges later needs to do a rebase)

Reply:
yep.  Wanted to call out that this isn't added by this PR so "don't look here" =D